### PR TITLE
Import file validation

### PR DESCRIPTION
**Pull request by n0rdy**

### CODE REVIEW

File: registry-manager-service-impl/src/main/java/de/adorsys/registry/manager/service/validator/property/AdapterIdValidator.java [hunk 1]
```diff
@@ -0,0 +1,23 @@
+package de.adorsys.registry.manager.service.validator.property;
+
+import de.adorsys.registry.manager.service.model.AspspBO;
+import de.adorsys.registry.manager.service.model.AspspValidationReportBO;
+
+import java.util.regex.Pattern;
+
+public class AdapterIdValidator implements PropertyValidator {
+    private static final String ADAPTER_ID_REGEX = ".{2,}-adapter";
```
Comment (berezkin88) [inline registry-manager-service-impl/src/main/java/de/adorsys/registry/manager/service/validator/property/AdapterIdValidator.java]: Recalling my conversation with @spetrychenko, AdapterId should consist of letters, numbers, and the hyphen only. If it's still true, I'd recommend adjusting the regexp rule to be compliant with that constraint.  Also, should `adapter` be always at the rear?

File: registry-manager-service-impl/src/main/java/de/adorsys/registry/manager/service/validator/property/BankNameValidator.java [hunk 2]
```diff
@@ -0,0 +1,13 @@
+package de.adorsys.registry.manager.service.validator.property;
+
+import de.adorsys.registry.manager.service.model.AspspBO;
+import de.adorsys.registry.manager.service.model.AspspValidationReportBO;
+
+public class BankNameValidator implements PropertyValidator {
+    private static final String EMPTY_PROPERTY_VALIDATION_ERROR_MESSAGE = "Bank name is empty";
+
```
Comment (berezkin88) [inline registry-manager-service-impl/src/main/java/de/adorsys/registry/manager/service/validator/property/BankNameValidator.java:13]: We have a constraint on the front that Banc Name should consist of word characters, whitespace characters, + german special symbols. I believe such a rule should be applied here as well.

File: registry-manager-service-impl/src/main/java/de/adorsys/registry/manager/service/validator/property/AdapterIdValidator.java (hunk 1 above)
Comment (n0rdy) [reply registry-manager-service-impl/src/main/java/de/adorsys/registry/manager/service/validator/property/AdapterIdValidator.java]:
```
AdapterId should consist of letters, numbers, and the hyphen only
```
Good point, thanks, I'll adjust my regex.

```
Also, should adapter be always at the rear?
```
At least we have such a constraint in `xs2a-adapter` app during the new adapter generation. So it seems to me, that we should be compliant with its rules.

File: registry-manager-service-impl/src/main/java/de/adorsys/registry/manager/service/validator/property/BankNameValidator.java (hunk 2 above)
Comment (n0rdy) [reply registry-manager-service-impl/src/main/java/de/adorsys/registry/manager/service/validator/property/BankNameValidator.java:13]: Yeah, why not, I'll do that, thanks

File: registry-manager-service-impl/src/main/java/de/adorsys/registry/manager/service/validator/property/AdapterIdValidator.java (hunk 1 above)
Comment (berezkin88) [reply registry-manager-service-impl/src/main/java/de/adorsys/registry/manager/service/validator/property/AdapterIdValidator.java]: `At least we have such a constraint in xs2a-adapter`

got it, I'll adjust validation on the front

**Review by spetrychenko — APPROVED** [review]

File: registry-manager-service-impl/src/main/java/de/adorsys/registry/manager/service/validator/property/AdapterIdValidator.java (hunk 1 above)
Comment (n0rdy) [reply registry-manager-service-impl/src/main/java/de/adorsys/registry/manager/service/validator/property/AdapterIdValidator.java]: Done, please, take a look

**Review by berezkin88 — APPROVED** [review]

**Pull request merged**